src/FSharp.SystemTextJson/Record.fs

```diff
@@ -205,8 +205,11 @@ type JsonRecordConverter<'T> internal (options: JsonSerializerOptions, fsOptions
         ctor fields :?> 'T
 
     override this.Write(writer, value, options) =
-        writer.WriteStartObject()
-        this.WriteRestOfObject(writer, value, options)
+        if isNull (box value) then
+            JsonSerializer.Serialize(writer, null, options)
```

Comment:
I think `writer.WriteNullValue()` would be both clearer and more efficient.

Reply:
Agreed, done